README.md

```diff
@@ -63,6 +63,14 @@ Download the latest A-4E-C installation file from the [releases page](https://gi
 
 Unzip the contents of the zip file into the following folder: `C:\Users\username\Saved Games\DCS`
 
```

Comment:
The ZIP-file has an extra folder `Community.A-4E-C-v1.3.1` that messes up the installation if the user literally extracts the contents of the file. Some users do read and follow instructions literally, while others might not realize the extra folder.

Add a comment about extracting the "MODS" directory in the Saved Games\DCS ?

Reply:
Valid point, the best way to avoid this might be to remove the folder from the zip, directly presenting the Mods folder when opening the zip. I'll see if this can be done in the build process. That's the neatest way to avoid that.

Other option is to just say something along the lines of: open the folder inside, which has a name similar to A-4E-Cv1.4 to present the Mods folder and Readme, then extract the Mods folder contents into the C:\Users......

If you've got a better formatting of that sentence feel free to put it up here. I'm wanting to keep it version agnostic and as simple as possible.

Reply:
I've spoken with Heclak and future releases will open directly to the mods folder, so this is a non-issue now. This pull request will need to wait until v1.4 to be released for the updated zips.